### PR TITLE
switch to industrial_ci

### PR DESCRIPTION
**Pull request by ipa-mdl**
This adds support for lunar
supersedes #8

### CODE REVIEW

**Comment by ipa-mdl** [comment]
I will squash it later (or you could squash-merge).
Downstream tests will be run as soon as  https://github.com/ros-industrial/industrial_ci/pull/137  was merged.

File: .travis.yml [hunk 1]
```diff
@@ -1,47 +1,30 @@
 # Travis Continuous Integration Configuration File For ROS Control Projects
 # Author: Dave Coleman
```
Comment (davetcoleman) [inline .travis.yml]: since you are switching to industrial_ci, you can remove me as author

File: .travis.yml [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
-compiler:
-  - gcc
+sudo: required
+dist: trusty
+language: generic
+
 notifications:
   email:
     recipients:
       - davetcoleman@gmail.com
```
Comment (davetcoleman) [inline .travis.yml]: remove me here too, please

File: .travis.yml [hunk 3]
```diff
@@ -1,47 +1,30 @@
...
-  - catkin_make -j2
+
+env:
+  global:
+    - UPSTREAM_WORKSPACE=https://raw.github.com/ros-controls/ros_control/kinetic-devel/ros_control.rosinstall
+    - ROS_PARALLEL_TEST_JOBS=-j1
+  matrix:
+    - ROS_DISTRO=kinetic ROS_REPO=ros
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO=lunar   ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=debian
```
Comment (davetcoleman) [inline .travis.yml]: why not add non-shadow-fixed for lunar also?

**Comment by ipa-mdl** [comment]
@davetcoleman: I have removed you from the notifcations.
@bmagyar: I have added the same notifications as for ros-controls/ros_control.

**Comment by bmagyar** [comment]
Great, thank you!

File: .travis.yml [hunk 4]
```diff
@@ -1,47 +1,30 @@
...
+  global:
+    - UPSTREAM_WORKSPACE=https://raw.github.com/ros-controls/ros_control/kinetic-devel/ros_control.rosinstall
+    - ROS_PARALLEL_TEST_JOBS=-j1
+  matrix:
+    - ROS_DISTRO=kinetic ROS_REPO=ros
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO=lunar   ROS_REPO=ros-shadow-fixed
+matrix:
+  allow_failures:
+    - env: ROS_DISTRO=lunar   ROS_REPO=ros-shadow-fixed
```
Comment (bmagyar) [inline .travis.yml]: Do we really want to allow failures? I think it's better not

File: .travis.yml [hunk 5]
```diff
@@ -1,47 +1,30 @@
...
-  - catkin_make -j2
+
+env:
+  global:
+    - UPSTREAM_WORKSPACE=https://raw.github.com/ros-controls/ros_control/kinetic-devel/ros_control.rosinstall
+    - ROS_PARALLEL_TEST_JOBS=-j1
+  matrix:
+    - ROS_DISTRO=kinetic ROS_REPO=ros
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed
+    - ROS_DISTRO=lunar   ROS_REPO=ros-shadow-fixed
```
Comment (bmagyar) [inline .travis.yml]: Could you please also add regular repo for Lunar?

**Comment by ipa-mdl** [comment]
I have updated the config, please squash-merge.

**Pull request merged**